ocaml/libs/xapi-compression/xapi_compression.ml

```diff
@@ -20,6 +20,12 @@ module type COMPRESSOR = sig
 
   (* Experimental decompressor which is fed from an fd and writes to a pipe *)
   val decompress_passive : Unix.file_descr -> (Unix.file_descr -> 'a) -> 'a
+
+  val compress_file :
```

Comment:
We can expect this to raise an exception if the file can't be found or written. Would be good to document it.